store.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Karma</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+
+  <body>
+    <header class="header">
+      <img src="img/karma-logo.svg" alt="Karma Logo" class="nav-logo" />
+      <nav>
+        <ul class="nav-list">
+          <li class="nav-item-meet-karma">
+            <a class="nav-link-meet-karma" href="index.html">Meet Karma</a>
+          </li>
+          <li>
+            <a href="#">How it Works</a>
+          </li>
+          <li>
+            <a href="store.html">Store</a>
+          </li>
+          <li>
+            <a href="#">Blog</a>
+          </li>
+          <li>
+            <a href="#">Help</a>
+          </li>
+          <li>
+            <a href="">Login</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section id="store-section">
+        <form class="form-container">
+          <p>Order your Karma wifi device today!</p>
+
+          <fieldset class="name">
+            <div class="name-group">
+              <label for="firstname">First name *</label>
+              <input id="firstname" type="text" name="username" />
+            </div>
+            <div class="name-group">
+              <label for="lastname">Last name *</label>
+              <input id="lastname" type="text" name="lastname" />
+            </div>
+          </fieldset>
+
+          <fieldset class="address">
+            <div class="address1">
+              <label for="address">Address *</label>
+              <input id="address" type="text" name="address" />
+            </div>
+            <div class="address2">
+              <label for="address2">Address 2</label>
+              <input id="address2" type="text" name="address2" />
+            </div>
+          </fieldset>
+
+          <fieldset class="city">
+            <div class="city-group">
+              <label for="city">City *</label>
+              <select id="city" name="city">
+                <option id="city-placeholder" value="city-placeholder" selected>
+                  Select your city...
```

Comment:
Should be in italic, according to [the design](https://www.figma.com/file/BPtLAPr1HhwQdNfxGyFouo/Karma-design?node-id=858%3A3)